ml/src/main/scala/ai/h2o/sparkling/ml/algos/regression/H2ORegressor.scala

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Maybe I looked at wrong places, I couldn't find any usage of this class

Reply:
It's used by generated classes. See https://github.com/h2oai/sparkling-water/pull/2211/files#diff-881337a409b96c7e9e252b23255d55a1R36